kafka-event-listener/src/main/java/org/hypertrace/core/kafka/event/listener/KafkaLiveEventListener.java

```diff
@@ -30,20 +30,29 @@
  * for sample usage and test. Note that testing requires Thread.sleep > poll timeout in between
  */
 public class KafkaLiveEventListener<K, V> implements AutoCloseable {
+  Queue<BiConsumer<? super K, ? super V>> callbacks;
```

Comment:
can this be private final? Also suggest making it a Collection - the queue is an impl detail that could just as easily be satisfied with a different collection implementation.